sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/administration/ServiceBusAdministrationAsyncClient.java

```diff
@@ -1365,6 +1367,65 @@ public Mono<Response<TopicProperties>> updateTopicWithResponse(TopicProperties t
         return withContext(context -> updateTopicWithResponse(topic, context));
     }
 
+    /**
+     * Creates a subscription with a default rule using {@link CreateSubscriptionOptions} and {@link CreateRuleOptions}.
+     *
+     * @param topicName Name of the topic associated with subscription.
+     * @param subscriptionName Name of the subscription.
+     * @param ruleName Name of the default rule the subscription should be created with.
+     * @param subscriptionOptions Information about the subscription to create.
+     * @param ruleOptions A {@link CreateRuleOptions} object describing the default rule.
+     *                    If null, then pass-through filter will be created.
+     *
+     * @return A Mono that completes with information about the created subscription.
+     * @throws ClientAuthenticationException if the client's credentials do not have access to modify the
+     *     namespace.
+     * @throws HttpResponseException If the request body was invalid, the quota is exceeded, or an error occurred
+     *     processing the request.
+     * @throws IllegalArgumentException if {@code topicName} or {@code subscriptionName} are null or empty strings.
+     * @throws NullPointerException if {@code subscriptionOptions} is null.
+     * @throws ResourceExistsException if a subscription exists with the same topic and subscription name.
+     * @see <a href="https://docs.microsoft.com/rest/api/servicebus/update-entity">Create or Update Entity</a>
+     */
+    @ServiceMethod(returns = ReturnType.SINGLE)
+    public Mono<SubscriptionProperties> createSubscription(String topicName, String subscriptionName, String ruleName,
+                                                           CreateSubscriptionOptions subscriptionOptions,
+                                                           CreateRuleOptions ruleOptions) {
+
+        return createSubscriptionWithResponse(topicName, subscriptionName, ruleName, subscriptionOptions, ruleOptions)
+            .map(Response::getValue);
+    }
+
+    /**
+     * Creates a subscription with default rule and returns the created subscription in addition to the HTTP response.
+     *
+     * @param topicName Name of the topic associated with subscription.
+     * @param subscriptionName Name of the subscription.
+     * @param ruleName Name of the default rule the subscription should be created with.
+     * @param subscriptionOptions Information about the subscription to create.
```

Comment:
it would be nice, if we can align this doc format with the one for ruleOptions, e.g. "A {@link CreateSubscriptionOptions} object describing the subscription properties"

Reply:
Same comment for sync API and createSubscriptionWithResponse

Reply:
Done